storage/pebble/registers.go

```diff
@@ -138,13 +138,13 @@ func (s *Registers) Store(
 }
 
 // LatestHeight Gets the latest height of complete registers available
-func (s *Registers) LatestHeight() uint64 {
-	return s.latestHeight.Load()
+func (s *Registers) LatestHeight() (uint64, error) {
```

Comment:
This would never return any error, because on startup, we must be able to read the latest height. And afterward, if storing registers was successful, the latest height must have been updated and can be retrieved without error.